test/cpp/end2end/hybrid_end2end_test.cc

```diff
@@ -540,7 +540,7 @@ TEST_F(HybridEnd2endTest, AsyncRequestStreamResponseStream) {
 }
 
 // Add a second service with one sync method.
-TEST_F(HybridEnd2endTest, AsyncRequestStreamResponseStream_SyncDupService) {
+TEST_F(HybridEnd2endTest, asyncRequestStreamResponseStreamSyncDupService) {
```

Comment:
I'm curious about the different uses of CamelCase and lowerCamelCase in test names, I see both in this PR. Is TEST_F expected to use lowerCamelCase, while TEST uses CamelCase? If so, I don't see this in the clang tidy documentation.

Reply:
Oh this is my mistake. It should start with a capital letter. Thanks for catching this. I just changed all other cases.